### ISSUE TITLE
[Package]: fcitx5-chinese-addons fcitx5-hangul

### ISSUE THREAD

**Issue by fromyb** [termux/termux-packages#17422]
### Why is it worth to add this package?

enable Chinese and Korean input for `fcitx5`

there is source code
- [fcitx5-chinese-addons](https://github.com/fcitx/fcitx5-chinese-addons)
- [fcitx5-hangul](https://github.com/fcitx/fcitx5-hangul)

### Source code URL

https://github.com/fcitx5

### Packaging policy acknowledgement

- [X] I certify that I have read [Termux Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy) and understand that my request will be denied in case of violation.

**Comment by twaik** [comment]
What for? In case of VNC server it is not needed because VNC client sends correct keycodes. In the case of termux-x11 it is not needed too.

**Comment by fromyb** [comment]
for termux-x11, use xfce4 direct with external keyboard

**Comment by fromyb** [comment]
@twaik any updates?

**Comment by twaik** [comment]
Any updates for what exactly? Both termux-x11 and vnc server have workarounds for Unicode input so it is not needed.

**Comment by fromyb** [comment]
em.. in my case(most phone made in China), my host system android on desktop mode can only type English(asciicode) by external keyboard even through with Gboard. so I use termux with proot-distro and install fcitx5 fcitx5-chinese-addons fcitx5-hangul for input Chinese and Korean to make it work. fcitx5 is already include in pkg, could you plz add fcitx5-chinese-addons fcitx5-hangul for fix it at direct use xfce4 without proot-distro ?

**Comment by twaik** [comment]
No need to ping me, I see messages in all termux repos.
You can use text bar to input custom text or to paste clipboard contents. Just swipe additional keyboard to the right.

**Comment by fromyb** [comment]
> No need to ping me, I see messages in all termux repos. You can use text bar to input custom text or to paste clipboard contents. Just swipe additional keyboard to the right.

yeah, but my wish is use android as desktop. with termux it cames true, so i don't have touch screen. in my case I only have external mouse and external keyboard, android phone is using like pc host with a 27 inch monitor. so I can't or hard to do some gesture like swipe. If those packages can add to termux repo, I can use termux-x11 as desktop like regular linux pc computer

**Comment by fromyb** [comment]
> No need to ping me, I see messages in all termux repos. You can use text bar to input custom text or to paste clipboard contents. Just swipe additional keyboard to the right.

thanks for anser, as your say, in termux we can use additional keyboard that way.
but when in termux-x11, it doesn't work.
in my case, termux-x11 is not only a vnc server, it can use xfce4 in x11 directly, so x11 is more than a vnc server, almost like a desktop environment. [fcitx5-chinese-addons](https://github.com/fcitx/fcitx5-chinese-addons) and [fcitx5-hangul](https://github.com/fcitx/fcitx5-hangul) is use for input with xfce4 directly install at termux-x11.
and I trust there is many people like me wait for that packages

**Comment by twaik** [comment]
> x11 is more than a vnc server, almost like a desktop environment.

Termux:X11 is **only** a vnc server, with some Android-related behaviour.

**Comment by fromyb** [comment]
correctly, I mean these packages should be contains in x11-repo, even use vnc server with termux

**Comment by fromyb** [comment]
and also for now, I find noway for solve with hardware keyboard in desktop environment like xfce4

**Comment by robertkirkman** [comment]
`pkg install fcitx5-chinese-addons fcitx5-hangul` now available
